packages/web/components/alert/toast.tsx

```diff
@@ -75,7 +79,11 @@ const ErrorToast: FunctionComponent<Alert> = ({ message, caption }) => (
     </div>
     <div className="text-white-high">
       <h6 className="mb-2 text-lg md:text-base">{t(message)}</h6>
-      {caption && <p className="text-sm md:text-xs">{t(caption)}</p>}
+      {caption && (
+        <p className="text-sm md:text-xs">
+          {typeof caption === "string" ? t(caption) : t(...caption)}
```

Comment:
how does this work? What data structure is `caption`?

Reply:
It's either a string or an array containing params to `t()`. It will be a string in the case it's an unexpected error string from the chain, in which case it will just be displayed. Otherwise if it's an array, the error string from the chain matched some regex and was converted into a translation string, that may contain a string localization parameter (like the name of a token, or amount of a token).